login-workflow/src/new-architecture/screens/ForgotPasswordScreen/ForgotPasswordScreenBase.test.tsx

```diff
@@ -18,23 +20,25 @@ describe('Forgot Password Screen Base', () => {
 
     it('renders without crashing', () => {
         render(
-            <ForgotPasswordScreenBase
-                WorkflowCardHeaderProps={{ title: 'Forgot Password' }}
-                WorkflowCardInstructionProps={{
-                    instructions: 'Forgot Password instructions',
-                }}
-                initialEmailValue={'test@email.net'}
-                WorkflowCardActionsProps={{
-                    showNext: true,
-                    nextLabel: 'Next',
-                    canGoNext: true,
-                    showPrevious: true,
-                    previousLabel: 'Back',
-                    canGoPrevious: true,
-                    currentStep: 2,
-                    totalSteps: 6,
-                }}
-            />
+            <AuthContextProvider {...authContextProps}>
```

Comment:
I had to wrap this using AuthContextProvider, because we are using `navigate` in the default success screen, which is coming from `useAuthContext` hook.
https://github.com/etn-ccis/blui-react-workflows/blob/0532e47e376d83ba7f2f4524b6674c4d9eb1b43a/login-workflow/src/new-architecture/screens/ForgotPasswordScreen/ForgotPasswordScreenBase.tsx#L18